client-app/core/utilities/helpers/index.ts

```diff
@@ -5,3 +5,11 @@ export function sleep(ms: number, resolvedValue?: any): Promise<any> {
 export function truncate(str: string, length: number): string {
   return str.length > length ? `${str.slice(0, length)}...` : str;
 }
+
+export function isObjectEmpty(object: Record<string, unknown>): boolean {
+  for (const property in object) {
```

Comment:
Isn't it better to use `return Object.keys(object).length === 0` instead?

Reply:
It's a matter of taste

Reply:
TS style.